### PR TITLE
WRQ-20241: Fix `QuickGuidePanels` to export `Panel` instead of `QuickGuidePanel`

### PR DESCRIPTION
**Pull request by juwonjeong**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
In https://github.com/enactjs/sandstone/pull/1567, we've renamed `Panel` of `QuickGuidePanels` to `QuickGuidePanel` by mistake.
But `QuickGuidePanels` need to export `Panel` and `QuickGuidePanels` as sandstone 2.9.0.alpha.1

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fixed `QuickGuidePanel` to `Panel`.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-20241

### Comments
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
Do we need to add a changelog for this?

**Comment by juwonjeong** [comment]
> Do we need to add a changelog for this?

I added changelog for this change

File: QuickGuidePanels/index.js [hunk 1]
```diff
@@ -15,19 +15,19 @@
  */
```
Comment (MikyungKim) [inline QuickGuidePanels/index.js:15]: Please also fix `QuickGuidePanel` at line 13.

Comment (juwonjeong) [reply QuickGuidePanels/index.js:15]: It's fixed.

**Pull request merged**